crates/task-impls/src/quorum_proposal.rs

```diff
@@ -464,7 +467,7 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>> QuorumProposalTaskState<TYPE
                 self.create_dependency_task_if_new(view, event_receiver, event_sender, event);
             }
             HotShotEvent::QuorumProposalRecv(proposal, _sender) => {
-                let view = proposal.data.get_view_number();
+                let view = proposal.data.get_view_number() + 1;
```

Comment:
@shenkeyao Instead of `<=` I realized that in the consensus task that things operator over `view + 1`, so that *should* cover the issue here.

Reply:
I think I don't understand the + 1 here.  If we see a proposal from view 5 then get a proposal for view 4, then: `view < self.latest_proposed_view` => `4 + 1 < 5`, which is false and we'll still process the proposal from view for despite it being old.